watchtower-core/src/main/resources/db/migration/V028__API_Integration_Add_Register_Columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE integration_entity ADD COLUMN register_state varchar(255) NOT NULL;
```

Comment:
won't this have a problem since the default will be NULL?

Reply:
Default for the state is NOT_SUPPORTED, the error is NULL though